cdk/domino_cdk/agent.py

```diff
@@ -69,18 +81,17 @@ def generate_install_config(
             },
         },
         "internal_docker_registry": {
+            "enabled": True,
```

Comment:
not a thing

Reply:
While you're correct and I've removed it, this inexplicably doesn't cause a validation error.

Reply:
We're still temporary lenient on this specific value until https://github.com/cerebrotech/platform-apps/pull/5830 is merged:
https://github.com/cerebrotech/platform-apps/blob/develop/src/domino_config/types1_1.py#L641